lib/bundles/inspec-compliance/cli.rb

```diff
@@ -18,78 +18,32 @@ def self.subcommand_prefix
       desc: 'Explicitly allows InSpec to perform "insecure" SSL connections and transfers'
     option :user, type: :string, required: false,
-      desc: 'Chef Compliance Username'
+      desc: 'Username'
     option :password, type: :string, required: false,
-      desc: 'Chef Compliance Password'
-    option :apipath, type: :string, default: '/api',
-      desc: 'Set the path to the API, defaults to /api'
+      desc: 'Password'
     option :token, type: :string, required: false,
-      desc: 'Chef Compliance access token'
+      desc: 'Access token'
     option :refresh_token, type: :string, required: false,
       desc: 'Chef Compliance refresh token'
-    def login(server) # rubocop:disable Metrics/AbcSize
-      # show warning if the Compliance Server does not support
-
-      options['server'] = server
-      url = options['server'] + options['apipath']
-
-      if !options['user'].nil? && !options['password'].nil?
-        # username / password
-        _success, msg = login_username_password(url, options['user'], options['password'], options['insecure'])
-      elsif !options['user'].nil? && !options['token'].nil?
-        # access token
-        _success, msg = store_access_token(url, options['user'], options['token'], options['insecure'])
-      elsif !options['refresh_token'].nil? && !options['user'].nil?
-        # refresh token
-        _success, msg = store_refresh_token(url, options['refresh_token'], true, options['user'], options['insecure'])
-        # TODO: we should login with the refreshtoken here
-      elsif !options['refresh_token'].nil?
-        _success, msg = login_refreshtoken(url, options)
-      else
-        puts 'Please run `inspec compliance login SERVER` with options --token or --refresh_token, --user, and --insecure or --not-insecure'
-        exit 1
-      end
-
-      puts '', msg
-    end
-
-    desc "login_automate SERVER --insecure --user='USER' --ent='ENT' --usertoken='TOKEN'", 'Log in to an Automate SERVER'
-    long_desc <<-LONGDESC
-      `login_automate` allows you to use InSpec with Chef Automate
-
-      You need to a user-token for communication with Chef Automate. More
-      information about token retrieval for Automate is available at:
-      https://docs.chef.io/api_delivery.html
-    LONGDESC
-    option :dctoken, type: :string,
+    option :dctoken, type: :string, required: false,
       desc: 'Data Collector token'
-    option :usertoken, type: :string,
-      desc: 'Automate user token'
-    option :user, type: :string,
-      desc: 'Automate username'
-    option :ent, type: :string,
+    option :ent, type: :string, required: false,
       desc: 'Enterprise for Chef Automate reporting'
-    option :insecure, aliases: :k, type: :boolean,
-      desc: 'Explicitly allows InSpec to perform "insecure" SSL connections and transfers'
-    def login_automate(server) # rubocop:disable Metrics/AbcSize
+    def login(server) # rubocop:disable Metrics/AbcSize
```

Comment:
You don't need the rubocop disable flag here anymore (when you refactored this out, the method became much simpler).    You can run a rubocop scan with 
```bundle exec rake lint```

Reply:
Great find! Fixed.

Reply:
Agreed, nice catch! Fixed.